orval.config.ts

```diff
@@ -98,5 +98,31 @@ export default defineConfig({
         indent: 2
       }
     }
+  },
+  dplCmsApi: {
+    output: {
+      mode: "split",
+      target: "src/core/dpl-cms-api/dpl.ts",
```

Comment:
Consider if this should be `cms` or `dpl-cms` instead of `dpl`.

It is the CMS we are interacting with not DPL :smile:

Reply:
👍